package.yaml

```diff
@@ -39,6 +39,7 @@ dependencies:
 
 default-extensions:
   - DeriveGeneric
+  - DeriveAnyClass
```

Comment:
I'd suggest turning this on only in the modules that need it. It can have weird action-at-a-distance otherwise.